brain/detail/util.h

```diff
@@ -0,0 +1,66 @@
+/* Copyright (c) 2013-2018, EPFL/Blue Brain Project
+ *                          Juan Hernando <jhernando@fi.upm.es>
+ *
+ * This file is part of Brion <https://github.com/BlueBrain/Brion>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#ifndef BRAIN_DETAIL_UTIL
+#define BRAIN_DETAIL_UTIL
+
+#include <random>
+#include <set>
+
+namespace brain
+{
+namespace
+{
+template <typename T>
+inline void _shuffle(T& container, const size_t* const seed)
+{
+    std::random_device randomDevice;
+    std::mt19937_64 randomEngine(randomDevice());
+    const char* seedEnv = getenv("BRAIN_CIRCUIT_SEED");
```

Comment:
Can we remove this special environment variable?

Reply:
I didn't add it. Any comment, @tribal-tec?

Reply:
If it's so annoying to people, it can be removed. Commandline it is then I guess? Documentation/scripts which use the env variable shall be adapted accordingly then.

Reply:
I doesn't annoy me at all. If there's anything depending on it, I wouldn't remove it.